nc/include/NcPhysics.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "physics/LayerMask.h"
+#include "physics/IClickable.h"
+
+namespace nc::physics
+{
+    void NcRegisterClickable(IClickable* clickable);
+    void NcUnregisterClickable(IClickable* clickable);
+    void NcRaycastToIClickables(LayerMask mask = LAYER_MASK_ALL);
```

Comment:
I don't usually see an interface marker in the name of a method. I would change to NcRaycastToClickables